src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -367,11 +379,13 @@ interface IOptionSettlementEngine {
     /**
      * @notice Gets information about the ERC20 token positions represented by a tokenId.
      * @param tokenId The token id for which to retrieve the Underlying position.
-     * @return underlyingPositions The Underlying struct for the supplied tokenId.
+     * @return underlyingPosition The Underlying struct for the supplied tokenId.
```

Comment:
I think `underlyingPositions` might be more understandable although still not very precise, bc we're already overloading "underlying" as a word, and making it plural might help convey it includes both the underlying asset and the exercise asset.

In fact, renaming `underlying` to `position` in order to de-overload "underlying" word might be better.